docs/en/02_Developer_Guides/03_Forms/Field_types/03_HTMLEditorField.md

```diff
@@ -224,9 +224,9 @@ Example: Remove field for "image captions"
 
 	:::php
 	// File: mysite/code/MyToolbarExtension.php
-	class MyToolbarExtension extends Extension {
+	class MyToolbarExtension extends DataExtension {
```

Comment:
I think `Extension` is the correct class here, `DataExtension` (while it would work fine) is more intended for `DataObject`.